tests/kuttl/tests/ovn_nicmappings/03-nicmappings-patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 60
+commands:
+  - script: |
+      oc patch OVNController -n $NAMESPACE ovncontroller-sample --type='json' -p='[{
+        "op": "replace",
+        "path": "/spec/nicMappings",
+        "value":{"datacentre2":"ospbr2"}
+      }]'
```

Comment:
we should also extend the test cases to cover the use case(nic mappings update and cleanup of old bridges/ports/enteries etc) described in https://issues.redhat.com//browse/OSPRH-3124 
Also test case for cleanup when nic Mappings are removed